assets/catalog/parsed_plugin_list.js

```diff
@@ -313,6 +343,18 @@ myfcn(
     "url": "https://github.com/intelsdi-x/snap-plugin-processor-tag"
   },
   {
+    "name": "Cassandra",
+    "type": "Publisher",
+    "description": "No description available.",
```

Comment:
Could we say something in the description like, ` "A publisher plugin for Cassandra cluster statistics"`